src/Listeners/UpdateLatestIndexDate.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Rapidez\Core\Listeners;
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Event;
+use Illuminate\Support\Facades\Storage;
+use Rapidez\Core\Events\IndexAfterEvent;
+
+class UpdateLatestIndexDate
+{
+    public function handle()
+    {
+        return Storage::disk('local')->put(
+            '/.last-index',
+            // With this we're just making sure the comparison
+            // is done within the same timezone in MySQL.
+            DB::selectOne('SELECT NOW() AS `current_time`')->current_time
```

Comment:
Because this is getting set specifically after indexing, if a product gets updated exactly during the command runtime, then it won't be caught. Would probably be an uncommon occurrence, but this seems fixable.

Reply:
Also, quick optimization:

```suggestion
            DB::scalar('SELECT NOW()')
```

Reply:
We wouldn't want to set the time before we've actually finished indexing in case indexing fails which is why we listen to the `IndexAfterEvent` instead of the `IndexBeforeEvent`

Reply:
I guess ideal would be to get the time when the indexing starts, but only write it when the indexing succeeds. Not sure if that can be done cleanly though.